Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml

```diff
@@ -3877,7 +3896,7 @@ inputs:
   playbookInputQuery:
 - key: BlockIndicatorsAutomatically
   value:
-    simple: "False"
+    simple: "True"
```

Comment:
should add the new cve to the playbook input "RelatedCVEs"

Reply:
Done